src/monmove.cpp

```diff
@@ -1298,7 +1298,8 @@ void monster::move()
             move_to( local_next_step, false, false, get_stagger_adjust( pos(), destination, local_next_step ) );
 
         if( !did_something ) {
-            moves -= 100; // If we don't do this, we'll get infinite loops.
+            // CHECKME: This used to always cost 100 moves, regardless of monster's speed!
```

Comment:
```suggestion
```
This should probably be a note in the description of the PR, a comment like this in the code is likely to simply live there forever never to be checked